src/implementing_new_features.md

```diff
@@ -204,3 +204,22 @@ tests/ui/feature-gates/ --bless`.
 [here]: ./stabilization_guide.md
 [tracking issue]: #tracking-issues
 [add-feature-gate]: ./feature-gates.md#adding-a-feature-gate
+
+## Call for testing
+
+Once the implementation is complete, the feature will be available to nightly users, but not yet part of stable Rust. This is a good time to write a blog post on [one of the Rust blogs](https://github.com/rust-lang/blog.rust-lang.org/) and issue a call for testing (here are three [example](https://blog.rust-lang.org/2021/08/03/GATs-stabilization-push.html) [blog](https://blog.rust-lang.org/inside-rust/2024/08/09/async-closures-call-for-testing.html) [posts](https://blog.rust-lang.org/2024/09/05/impl-trait-capture-rules.html) to give you the idea). The post should highlight how the feature works, what areas you'd like people to play with, and how they can supply feedback.
```

Comment:
There is also the [call-for-testing section](https://this-week-in-rust.org/blog/2025/01/22/this-week-in-rust-583/#calls-for-testing) in TWIR. I think some of us have been trying to use it more often[^1].

It might useful to mention it for feature that are maybe to small for a full scale blog post. Although from my experience it effect is minimal at best.

[^1]: https://github.com/rust-lang/rust/issues/131204#issuecomment-2569314526